guides/v3.5.0/tutorial/service.md

```diff
@@ -208,25 +156,30 @@ This property will be passed in to the component by its parent template below.
 
 Finally open the template file for our `rental-listing` component and add the new `location-map` component.
 
-```handlebars {data-filename="app/templates/components/rental-listing.hbs" data-diff="+20"}
+```handlebars {data-filename="app/templates/components/rental-listing.hbs" data-diff="+25"}
 <article class="listing">
-  <a {{action 'toggleImageSize'}} class="image {{if isWide "wide"}}">
-    <img src="{{rental.image}}" alt="">
+  <a
+    onclick={{action "toggleImageSize"}}
+    role="button"
+    class="image {{if isWide "wide"}}"
+  >
+    <img src={{rental.image}} alt="">
     <small>View Larger</small>
   </a>
-  <h3>{{rental.title}}</h3>
-  <div class="detail owner">
-    <span>Owner:</span> {{rental.owner}}
-  </div>
-  <div class="detail type">
-    <span>Type:</span> {{rental-property-type rental.category}}
-      - {{rental.category}}
-  </div>
-  <div class="detail location">
-    <span>Location:</span> {{rental.city}}
-  </div>
-  <div class="detail bedrooms">
-    <span>Number of bedrooms:</span> {{rental.bedrooms}}
+  <div class="details">
```

Comment:
I changed the DOM of this component to make formatting this with flexbox easier.  I had to make some tutorial css addon changes because the leaflet map required different styling. I still need to ripple this back through the component parts for the tutorial.